### PR TITLE
feat: add initial dockerfile and make commands

### PR DESCRIPTION
**Pull request by nickshine**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,47 @@
...
+release: clean compile package
+
+docker-build:
+	docker build -t $(PROJECT):$(VERSION) .
+
+docker-run:
+	docker run --rm -v $(shell pwd)/config.yaml:/config.yaml $(PROJECT):$(VERSION)
+
+docker-release: docker-build
+docker-release:
```
Comment (circa10a) [inline Makefile]: Don't forget to add the obligatory `latest` tag

Comment (circa10a) [reply Makefile]: Actually nvm. Forgot docker hub's auto build will do that.

**Comment by circa10a** [comment]
LGTM

Except this:
```
GOOS=windows GOARCH=amd64 go build $(BUILD_FLAGS) -o bin/windows/amd64/$(BINARY).exe
```

But what ya gonna do?

File: Makefile (hunk 1 above)
Comment (nickshine) [reply Makefile]: If you want to do Docker Hub's auto build, then wouldn't we not have any docker stuff in Travis CI?

Comment (circa10a) [reply Makefile]: Latest is pretty simple for docker hub to handle, but I don't like the way it versions the tag for you. For example on a push of tag `v1.0.0` docker hub would create `circa10a/go-aws-news:version-1.0.0`. I prefer the actual semantic version for the tag. Unless you feel differently.

**Pull request merged**